buy_together_app/models.py

```diff
@@ -1,3 +1,50 @@
-# from django.db import models
+from django.db import models
+from django.core.validators import MinLengthValidator
+from django.core.exceptions import ValidationError
 
-# Create your models here.
+
+class Product(models.Model):
+    qr_code = models.CharField(max_length=32, validators=[MinLengthValidator(32)], primary_key=True)
+    product_name = models.CharField(max_length=32, validators=[MinLengthValidator(1)])
+    description = models.TextField(validators=[MinLengthValidator(1)])
+
+    @staticmethod
+    def filter_qr(qr):
+        if qr:
+            return Product.objects.filter(qr_code=qr)
+        return Product.objects.all()
+
+    @staticmethod
+    def filter_name(nm):
+        if nm:
+            return Product.objects.filter(product_name=nm)
+        else:
+            return []
```

Comment:
You can write it in one line: `return Product.objects.filter(product_name=nm) if nm else []`
Although, I'm not sure why we need this method. Can't we just call `Product.objects.filter(product_name=nm)`?

Reply:
Its a method for the future to be avaible to use an already written function instead of writing the filter on objects over and over again and for the test
Done